### DISCUSSION TITLE
Long running jobs

### DISCUSSION THREAD

**Discussion by nik-ver** [loopbackio/loopback-next#7682] [Q&A]
Hello community,

i want to ask you if there is any feature in LB4 what i can use to run "long running jobs" (such as an invoice run). The idea is to start the jobs from the browser client and get a response like "Job started". Then i want to monitor the currently running jobs to see their progress / state and to abort them if i want. I thougt of using something like the node child_process, but maybe there is a Loopback way of doing this.

I am thankfull for any type of help.

**Comment by raymondfeng** [comment]
See https://loopback.io/doc/en/lb4/Running-cron-jobs.html
